src/components/table/Table.stories.tsx

```diff
@@ -0,0 +1,433 @@
+                             <th>Bar</th>
+                             <th>Baz</th>
+                         </tr>
+                     </thead>
+                     <tbody>
+                         <tr>
+                             <td className="left">Pig Latin</td>
+                             <td>Oofay</td>
+                             <td>Arbay</td>
+                             <td>Azbay</td>
+                         </tr>
+                         <tr>
+                             <td className="left">Bizarro</td>
+                             <td>Bar</td>
+                             <td>Foo</td>
+                             <td>Bang</td>
+                         </tr>
+                         <tr>
+                             <td className="left">Hodor</td>
+                             <td>Hodor</td>
+                             <td>Hodor</td>
+                             <td>Hodor</td>
+                         </tr>
+                     </tbody>
+                 </Table>
+             </div>
+             <div className="clr-col-12 clr-col-lg-6 clr-col-xl-4">
+                 <Table>
+                     <thead>
+                         <tr>
+                             <th className="left">Color</th>
+                             <th>R</th>
+                             <th>G</th>
+                             <th>B</th>
+                         </tr>
+                     </thead>
+                     <tbody>
+                         <tr>
+                             <td className="left">Black</td>
+                             <td>0</td>
+                             <td>0</td>
+                             <td>0</td>
+                         </tr>
+                         <tr>
+                             <td className="left">New Black</td>
+                             <td>255</td>
+                             <td>165</td>
+                             <td>0</td>
+                         </tr>
+                         <tr>
+                             <td className="left">Pantone 292</td>
+                             <td>98</td>
+                             <td>168</td>
+                             <td>229</td>
+                         </tr>
+                     </tbody>
+                 </Table>
+             </div>
+             <div className="clr-col-12 clr-col-lg-6 clr-col-xl-4">
+                 <Table>
+                     <thead>
+                         <tr>
+                             <th className="left">Name</th>
+                             <th className="left">Weakness</th>
+                         </tr>
+                     </thead>
+                     <tbody>
+                         <tr>
+                             <td className="left">Frankenstein</td>
+                             <td className="left">Fire</td>
+                         </tr>
+                         <tr>
+                             <td className="left">Dracula</td>
```

Comment:
Do we also have class for `right `align items ?

Reply:
No we do not have a class for right